src/Aquifer.API/Endpoints/Projects/AssignedToSelf/List/Endpoint.cs

```diff
@@ -40,13 +40,21 @@ private async Task<List<Response>> GetProjectsAsync(CancellationToken ct)
                         : null,
                 Counts = new ProjectResourceStatusCounts
                 {
-                    NotStarted = x.ProjectResourceContents.Count(prc => ProjectResourceStatusCounts.NotStartedStatuses.Contains(prc.ResourceContent.Status)),
-                    InProgress = x.ProjectResourceContents.Count(prc => ProjectResourceStatusCounts.InProgressStatuses.Contains(prc.ResourceContent.Status)),
-                    InManagerReview =
-                        x.ProjectResourceContents.Count(prc => ProjectResourceStatusCounts.InManagerReviewStatuses.Contains(prc.ResourceContent.Status)),
+                    AwaitingAiDraft = 
```

Comment:
Could we also rename this and the other occurrences of `AwaitingAiDraft` for project resource counts?